modules/ROOT/pages/hot-deployment.adoc

```diff
@@ -0,0 +1,27 @@
+= Application Hot Deployment
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: runtime, deploy, hot deployment, application, applications
+
+You can now modify your configuration files and custom classes and have them reloaded without having to restart Mule.
+
+== Outline of a Deployment
+
+Here is a quick summary for deploying an app 'foo':
+
+* Create a directory under `$MULE_HOME/apps/foo`
+* Place your packaged application (e.g. `foo.jar`) at `$MULE_HOME/apps`
+* Jar custom classes (if any), and put them under `$MULE_HOME/apps/foo/META-INF/mule-src/foo/src/main/java/`
+* Start your app with `mule -M-Dmule.deploy.applications=foo` (see xref:application-deployment[Application Deployment] section for more information).
+
+As a bonus, application's master config file is monitored, so if there are any class changes you want to pick up or simply modify the config, save or touch any of the mule configuration files and Mule will hot-reload the application.
```

Comment:
all application's config files are monitored, not only the master file

Reply:
```suggestion
As a bonus, application's configuration files are monitored, so if there are any class changes you want to pick up or simply modify any config, save or touch any of the mule configuration files and Mule will hot-reload the application.
```